tests/UI/Base.php

```diff
@@ -92,4 +93,24 @@ public function getDefaultRenderer() {
 	public function normalizeHTML($html) {
 		return trim(str_replace("\n", "", $html));
 	}
+
+	/**
+	 * @param string $expected_html_as_string
+	 * @param string $html_as_string
+	 */
+	public function assertHTMLEquals($expected_html_as_string,$html_as_string){
+		$html = new DOMDocument();
+		$html->formatOutput = true;
+		$html->preserveWhiteSpace = false;
+
+		$expected = new DOMDocument();
+		$expected->formatOutput = true;
+		$expected->preserveWhiteSpace = false;
+
+		$html->loadXML($this->normalizeHTML($html_as_string));
```

Comment:
Do we still need my homemade `normalizeHTML` or does `DOMDocument` do something similar for us?